src/datepicker/day.js

```diff
@@ -234,10 +245,12 @@ export default class Day<T = Date> extends React.Component<
       return false;
     }
     // fix flow by passing a specific arg type and remove 'Array.isArray(value)'
-    if (Array.isArray(value) && value.length > 1) {
+    if (Array.isArray(value) && value[0] && value[1]) {
       return this.dateHelpers.isDayInRange(
         this.clampToDayStart(date),
+        // $FlowFixMe -- value[0] can't be null or undefined
```

Comment:
I played around with this a bit but I can't figure out a good way to re-write the logic to obviate the need for the `FlowFixMe`s. Open to suggestions!

Reply:
I believe if you assign value[0] and value[1] to local variables, flow can make more assertions on whether the value changes or not. So maybe on L247 doing something like `const [start, end] = value;` would help out here and for the instances below

Reply:
Fantastic, thank you.